src/components/CourseChecklist/CourseChecklist.scss

```diff
@@ -2,15 +2,42 @@
 @import "~bootstrap/scss/_buttons";
 @import "~bootstrap/scss/utilities/_screenreaders";
 
-$light-color: theme-color("light");
 $lightest-color: theme-color("lightest");
 $primary-color: theme-color("primary");
 $success-color: theme-color("success");
 
+.assignment-list-item {
+  list-style: none;
+  display: inline-block;
+
+  &:after { 
```

Comment:
Making the list of links comma separated and addressing fenceposting.

Reply:
This is awesome - I didn't know you could do it this way.